dev-tools/iota-private-network/experiments/network-benchmark.sh

```diff
@@ -169,12 +185,62 @@ block_connection() {
   log "Blocked traffic $A → $B"
 }
 
-# restart a container with validator
+# Restart a validator container with configurable database handling.
+# Supports three modes:
+#   - preserve-consensus: Remove only authorities_db, keep consensus_db
+#   - full-reset: Remove both authorities_db and consensus_db
+#   - clean-restart: Don't remove any databases, clean docker restart only
```

Comment:
tbh, "clean restart" might be a bit confusing. For me a "clean" restart could be exactly the opposite, to remove all databases. Can't we call it "restart" or "simple-restart" ?